### PR TITLE
added workflows

### PR DESCRIPTION
**Pull request by henriterhofte**
in order to enable:
- building & deploying /docs/installation_manual;
- - building & deploying /docs/device_info page;
- signing secure firmware, I added the workflows to this repo

Could not test docs building & deployment without  a merge yet...

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Find documentation here:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# @energietransitie/twomes-maintainers will be requested for
+# a review when someone opens a pull request.
+# change the line below to suit your own organization
+*   @energietransitie/twomes-maintainers
```
Comment (n-vr) [inline .github/CODEOWNERS:7]: @henriterhofte can you give twomes-maintainers access to this repo, to resolve this issue?

**Review by n-vr — COMMENTED** [review]
I updatet the workflows. The rest looks great.

One more thing about the workflows, should we change the environment variable WINDESHEIM_ENERGIETRANSITIE_SIGNING_KEY to SECURE_BOOTLOADER_SIGNING_KEY?:
https://github.com/energietransitie/twomes-presence-detector/blob/42ece099a3842fde7d1d91580ee08e5674f13b85/.github/workflows/release.yml#L56-L59

File: .github/CODEOWNERS (hunk 1 above)
Comment (n-vr) [reply .github/CODEOWNERS:7]: I also did NOT push the updates to the workflows, since this means I don't have the rights to do that.

Comment (henriterhofte) [reply .github/CODEOWNERS:7]: I updated the access rights like you requested

Comment (n-vr) [reply .github/CODEOWNERS:7]: Thank you. That fixed the codeowners file and allowed my to push some commits.

Comment (henriterhofte) [reply .github/CODEOWNERS:7]: I added SECURE_BOOTLOADER_SIGNING_KEY as an environment secret to https://github.com/energietransitie/twomes-generic-esp-firmware/settings/secrets/actions... it seems not re-usable in other repos like this one. Did we decide to replicate the SECURE_BOOTLOADER_SIGNING_KEY in each repo for Twomes ESP32 firmware? I remember that organisational secrets were not usable for this...?

**Pull request merged**